modules/services/mopidy.nix

```diff
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.mopidy;
+
+in
+
+{
+  options = {
+    services.mopidy = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the Mopidy Daemon.";
+      };
+
+      package = mkOption {
+        type = types.path;
+        default = "/usr/local";
```

Comment:
I would prefer the package from nixpkgs here, you can do this in your configuration until it's fixed there.